src/cats/monad/maybe.cljc

```diff
@@ -304,14 +302,16 @@
   {:pre [(maybe? m)]}
   (if (nothing? m)
     (lazy-seq [])
-    (lazy-seq [(m/extract m)])))
+    (lazy-seq [(p/extract m)])))
 
-;; TODO: use a transducer when we support 1.7.0+
 (defn cat-maybes
-  "Given a collection of maybes, return a sequence of the values that the
-   just's contain."
+  "Given a collection of maybes, return a sequence of the values
+  that the just's contain."
   [coll]
-  (map m/extract (filter just? coll)))
+  (let [xform (comp
```

Comment:
Since it's not a stateful transducer we can even define it outside `cat-maybes`.